hypothesis-python/src/hypothesis/internal/conjecture/data.py

```diff
@@ -705,6 +737,7 @@ def __check_capacity(self, n):
     def conclude_test(self, status, interesting_origin=None):
         assert (interesting_origin is None) or (status == Status.INTERESTING)
         self.__assert_not_frozen("conclude_test")
+        self.__concluded = True
```

Comment:
This doesn’t seem to be used anywhere, even temporarily.

Reply:
Ah, it's not, thanks. From an earlier version of the code that got axed.